test/functional/rpc_createmultisig.py

```diff
@@ -217,6 +180,41 @@ def do_multisig(self, nkeys, nsigs, output_type, wallet_multi):
         txinfo = node0.getrawtransaction(tx, True, blk)
         self.log.info("n/m=%d/%d %s size=%d vsize=%d weight=%d" % (nsigs, nkeys, output_type, txinfo["size"], txinfo["vsize"], txinfo["weight"]))
 
+    def test_mixing_uncompressed_and_compressed_keys(self, node, wallet_multi):
+        self.log.info('Mixed compressed and uncompressed multisigs are not allowed')
+        pk0, pk1, pk2 = [getnewdestination('bech32')[0].hex() for _ in range(3)]
+
+        # decompress pk2
+        pk_obj = ECPubKey()
+        pk_obj.set(bytes.fromhex(pk2))
+        pk_obj.compressed = False
+        pk2 = pk_obj.get_bytes().hex()
+
+        # Check all permutations of keys because order matters apparently
+        for keys in itertools.permutations([pk0, pk1, pk2]):
+            # Results should be the same as this legacy one
+            legacy_addr = node.createmultisig(2, keys, 'legacy')['address']
+
+            if wallet_multi is not None:
+                # 'addmultisigaddress' should return the same address
+                result = wallet_multi.addmultisigaddress(2, keys, '', 'legacy')
+                assert_equal(legacy_addr, result['address'])
+                assert 'warnings' not in result
+
+            # Generate addresses with the segwit types. These should all make legacy addresses
+            err_msg = ["Unable to make chosen address type, please ensure no uncompressed public keys are present."]
```

Comment:
Nit: `err_msg_list`

Reply:
sure, will change it if I have to retouch the code.